.github/test-ci-trigger.md

```diff
@@ -0,0 +1 @@
+# Testing CI with updated token permissions
```

Comment:
Why is this file here?

Reply:
Good catch. Removed in c67cc2741c67a298b5cbca2e7f8da8fe57d44a86.